resources/css/filament/admin/theme.css

```
@@ -1,4 +1,220 @@
+  font-family: var(--font-sans), serif;
+}
+
+.dark .fi-panel-system {
+  background: #0d1117;
+}
+
+/* Topbar — match docs navbar (white, 60px, no shadow) */
+.fi-panel-system .fi-topbar {
+  background: #ffffff;
+  box-shadow: none;
+  /*border-bottom: 1px solid rgb(0 0 0 / 6%);*/
```

Stylelint (17.11.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)